docs/courses/README.md

```diff
@@ -1,28 +1,101 @@
-# Contributing to LibreLingo courses
+# LibreLingo Course Documentation
 
-LibreLingo courses consist of files that contain course content. These files
-are organized in a specific directory structure.
-[Learn how to edit or create courses](editing-courses.md)!
+This document describes how the course system is laid out on LibreLingo and information on how to contribute to the course data.
 
+This document is the first step whether you want to edit an existing course, contribute a translation to an existing course or create a new one from scratch. This is also a good resource to read if you just want to learn about the course system in general.
+
+---
+
+**NOTE**: If you are not clear about anything on this document feel free to post on [Discussions](https://github.com/kantord/LibreLingo/discussions) so that the community can help you further.
+
+---
+
+**Table of Contents:**
+- [Navigation](#navigation)
+- [Basics](#basics)
+  - [Terminology](#terminology)
+  - [Exploring the courses](#exploring-courses)
+  - [Things to do for new contributors](#things-new-contributors)
+- [Courses currently open for contributions](#courses-open)
+  - [For English speakers](#courses-from-english)
+- [Caution](#caution)
+  - [Languages that don't use the Latin alphabet](#non-latin-alphabet)
+
+## Navigation
+
+- If you are totally new to the project, read the [Basics](#basics) section below first. It will give you a general understanding about things.
+- If you want to edit existing courses or translate them, go to [this page](editing-courses.md).
+- If you want to create new courses (typically suited for course designers or advanced users), go to [this page](creating-courses.md).
+
+
+## Basics
+
+### Terminology
+
+If you want to work on courses you need to have an idea on some basic terminology. We have _Courses_, _Modules_ and _Skills_. In a nutshell:
+
+- A **Course** is a set of modules. There is typically one course for each language. e.g. `spanish-from-english`, `french-from-english`.
+- A **Module** is a set of lessons. They act like groups to keep lessons more organized.
+- A **Skill** is a lesson with a set of questions.
+
+So the hierarchy would be: `Courses > Modules > Skills`
+
+There are also languages to and from which users will learn.
+
+- **Destination language** is the language which you want to test for or teach.
+- **Target audience's language** is the language of the users for which the course is made.
+
+The course directory name is usually structured in `<destination language>-from-<target audience's language>`. e.g. `french-from-english` teaches French to English speakers.
```

Comment:
Is this part very relevant here? To me it seems it's more relevant when you are creating a new course

Reply:
I kept it to make it clear how the course directory names are structured. I thought it's important to know which language serves what purpose in the directory name.

Maybe we can move it to somewhere else? Maybe under the next heading "Exploring the courses" after the line "...you will find courses."?

Reply:
Yes, it might make sense under "exploring courses"